web/react/stores/user_store.jsx

```diff
@@ -64,12 +65,12 @@ var UserStore = assign({}, EventEmitter.prototype, {
   },
   setCurrentId: function(id) {
     if (id == null)
-      sessionStorage.removeItem("current_user_id");
+      localStorage.removeItem("current_user_id");
```

Comment:
These need to be localStorage because BrowserStore uses them as the prefix. Otherwise we get an infinite loop. 

I have not tested if this works with /team/ yet so there may be a few changes.